### PR TITLE
if left memory reaches threshold, also add a notification

### LINKED ISSUE #1269: Add the "low RAM warnings" to the Notification system
We alreayd have low RAM warnings in js-controller. We should provide these info also to the notification system because no user see them im logs, but we could prevent OOM situations before

### PR DESCRIPTION
**Pull request by foxriver76**
- fixes #1269

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -3292,11 +3298,20 @@ async function startInstance(id, wakeUp) {
...
+        if (availableMemMB < (typeof config.system.memLimitError === 'number' ? config.system.memLimitError : 50)) {
+            logger.error(`${hostLogPrefix} Your system has only ${availableMemMB} MB RAM left available and an additional adapter process is started. Please check your system, settings and active instances to prevent swapping and Out-Of-Memory situations!`);
+            logger.error(`${hostLogPrefix} In future versions, the adapter might not be started!`);
+        } else {
+            logger.warn(`${hostLogPrefix} Your system has only ${availableMemMB} MB RAM left available and an additional adapter process is started. Please check your system, settings and active instances to prevent swapping and Out-Of-Memory situations!`);
+        }
+
+        // add it to notifications for popup
+        try {
+            await notificationHandler.addMessage('system', 'memIssues', `Your system has only ${availableMemMB} MB RAM left available and an additional adapter process is started. Please check your system, settings and active instances to prevent swapping and Out-Of-Memory situations!`, `system.host.${hostname}`);
```
Comment (Apollon77) [inline main.js:3311]: Maybe (just to be sure) add a check that notificationHandler is defined? ;-)

Comment (foxriver76) [reply main.js:3311]: hm why its inside try catch

Comment (Apollon77) [reply main.js:3311]: ha ... ok... nice ... yes ok ... :-)

Comment (foxriver76) [reply main.js:3311]: lol ;-)

**Pull request merged**